docs/changelog.rst

```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 This project adheres to `Semantic Versioning <http://semver.org/>`_.
 Please note that the changes before version 1.10.0 have not been documented.
 
-v3.2.1
```

Comment:
Why don't we separate the changes? What actually changed between 3.2.1(0) and 3.2.2?

Reply:
Agreed, updated